api/src/main/java/com/nicknathanjustin/streamercontracts/contracts/dtos/PrivateContract.java

```diff
@@ -11,11 +11,15 @@
 @Data
 public class PrivateContract extends Contract {
 
-    private boolean userHasVoted;
+    private Boolean userVote;
     
     public PrivateContract(@NonNull final ContractModel contract) {
         super(contract);
-        Stream<VoteModel> userVotes = contract.getVotes().stream().filter(voteModel -> voteModel.getVoter().getTwitchUsername().equals(contract.getStreamer().getTwitchUsername()));
-        this.userHasVoted = userVotes.count() > 0;
+        this.userVote = null;
```

Comment:
what does a null value indicate here? Do we need null or can true/false capture all the information we need?

Reply:
I think i get it looking a second time. null is no vote, false is the user voted it wasnt complete, true user voted it was complete... is that right?